File: Java/Warmup-1/front3.java

public class warmup1 {

	// this function return a new string which is three copies of the front
	// front = three first chars
	public String front3(String str) {

		String s = "";
		
		if (str.length() < 3)
			s = str + str + str;
		else
			s = str.substring(0, 3) + str.substring(0, 3) + str.substring(0, 3); 
		
		return s;

	}
	
	public static void main(String[] args) {
		
		warmup1 o = new warmup1();
		
		System.out.println(o.front3("Java"));
		System.out.println(o.front3("Chocolate"));
		System.out.println(o.front3("abc"));

	}
	
}

File: Python/Warmup-1/front3.py

# this function return a new string which is three copies of the front
# front = three first chars
def front3(str):
	s = ""

	if len(str) < 3:
		s = str + str + str
	else:
		s = str[:3] + str[:3] + str[:3]

	return s

print(front3("Java"))
print(front3("Chocolate"))
print(front3("abc"))
